src/scene_battle_rpg2k3.cpp

```diff
@@ -2768,7 +2768,7 @@ void Scene_Battle_Rpg2k3::CBAInit() {
 	}
 
 	if (cba_action->GetCBAMovement() != lcf::rpg::BattlerAnimationItemSkill::Movement_none) {
-		sprite->SetAfterimageAmount(cba_action->GetWeaponAnimationData()->after_image == lcf::rpg::BattlerAnimationItemSkill::AfterImage_add ? 3 : 0);
+		sprite->SetAfterimageAmount(cba_action->GetCBAAfterImage() == lcf::rpg::BattlerAnimationItemSkill::AfterImage_add ? 3 : 0);
```

Comment:
I just noticed: This is incorrect in the liblcf enum. It is an Afterimage, not an AfterImage.
Will submit a patch to liblcf to fix this. Good moment right now, will not break anythign except the PR :)